### PR TITLE
New York Page Copy (Project 1)

### PR DESCRIPTION
**Pull request by zilton7**
Here I have finished building the copy of New York Times page, I have styled it and have done this following the linters guidance

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @zilton7  :wave:

__You've made laudable progress in this project.
However, some changes are required before they can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Always ensure that all the general requirements for each project are taken care of. For instance, it is mentioned in the general requirements for this project that `Files not related to the project should not be committed and added to .gitignore` (e.g. `.vscode`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Your project screenshot is not rendering in your Readme file. This can be a reference/addressing problem. Kindly fix it.
- [x] The `MIT license` and `Issues Page` links in your Readme are broken. 
- [x] The Getting Started, Prerequisites, Setup, Install, Usage, Run tests, Run tests and Deployment sections are empty. Kindly add contents to each of these sections. You can remove a section completely if such a section is not applicable to your project.
- [x] Also, ensure that your Pull Request (PR) has a meaningful title (e.g. New York Times) and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). You did not provide any description for your project PR. You should summarize what you've done in this PR as your project description.
- [x] You should make it a habit to always commit often with appropriate commit messages. Check [this article ](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) on how to compose descriptive commit messages.
- [x] You are not expected to merge your feature branch to the main/master branch until your project is approved. from your PR page, it shows that you've already merged. This is not the convention. Please take note.
- [x] Your page layout and stylings is totally different from the source. Kindly re-style your webpage so that it looks similar to the original page. For clarity, you're expected to build something similar to [this page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0), using combination of all three techniques of alignment (float, flex and grid).



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html [hunk 1]
```diff
@@ -299,7 +297,7 @@ <h3 class="title">The Theory of Inflation</h3>
                 been there 23 times, he said, wintering over in 1994. “I’ve been hooked ever since,” he said.
             </p>
             <div>
-                <img class="image" src="/img/18cosmos-2-superJumbo.webp" alt="">
+                <img class="image" src="./img/18cosmos-2-superJumbo.webp" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add a description in `alt` attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @zilton7 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Also, ensure that your Pull Request (PR) has a meaningful title (e.g. New York Times) and a brief description . You did not provide any description for your project PR. You should summarize what you've done in this PR as your project description.
- [x] Add a description in `alt` attribute
- [x] The NY Times logo should be a link. ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @zilton7 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**